### PR TITLE
APP-1161 Add solution content to error logging

### PR DESCRIPTION
**Pull request by robson-sanchez**

### CODE REVIEW

**Review by ardsilva — COMMENTED** [review]
LGTM

File: integration-commons/src/main/java/org/symphonyoss/integration/exception/config/ForbiddenUserException.java [hunk 1]
```diff
@@ -24,16 +24,8 @@
...
-  }
-
-  public ForbiddenUserException(String message) {
-    super(message);
-  }
-
-  public ForbiddenUserException(Throwable cause) {
-    super(cause);
+  public ForbiddenUserException(String message, Throwable cause, String... solutions) {
+    super(message, cause, solutions);
```
Comment (ardsilva) [inline integration-commons/src/main/java/org/symphonyoss/integration/exception/config/ForbiddenUserException.java:28]: We need to ensure that the constructors aren't used

File: integration-commons/src/main/java/org/symphonyoss/integration/exception/config/NotFoundException.java [hunk 2]
```diff
@@ -23,16 +23,8 @@
...
-  }
-
-  public NotFoundException(String message) {
-    super(message);
-  }
-
-  public NotFoundException(Throwable cause) {
-    super(cause);
+  public NotFoundException(String message, Throwable cause, String... solutions) {
+    super(message, cause, solutions);
```
Comment (ardsilva) [inline integration-commons/src/main/java/org/symphonyoss/integration/exception/config/NotFoundException.java:27]: We need to ensure that the constructors aren't used

File: integration-commons/src/main/java/org/symphonyoss/integration/exception/config/RemoteConfigurationException.java [hunk 3]
```diff
@@ -21,16 +21,7 @@
  */
 public class RemoteConfigurationException extends IntegrationConfigException {
 
-  public RemoteConfigurationException() {
-    super("Unable to retrieve configurations.");
+  public RemoteConfigurationException(String message, Throwable cause, String... solutions) {
+    super(message, cause, solutions);
```
Comment (ardsilva) [inline integration-commons/src/main/java/org/symphonyoss/integration/exception/config/RemoteConfigurationException.java:25]: We need to ensure that the constructors aren't used

File: integration-commons/src/main/java/org/symphonyoss/integration/exception/config/ForbiddenUserException.java (hunk 1 above)
Comment (crepache) [reply integration-commons/src/main/java/org/symphonyoss/integration/exception/config/ForbiddenUserException.java:28]: Constructor not used.

File: integration-commons/src/main/java/org/symphonyoss/integration/exception/config/NotFoundException.java (hunk 2 above)
Comment (crepache) [reply integration-commons/src/main/java/org/symphonyoss/integration/exception/config/NotFoundException.java:27]: Constructor not used

File: integration-commons/src/main/java/org/symphonyoss/integration/exception/config/RemoteConfigurationException.java (hunk 3 above)
Comment (crepache) [reply integration-commons/src/main/java/org/symphonyoss/integration/exception/config/RemoteConfigurationException.java:25]: Constructor not used

**Pull request merged**